### PR TITLE
Updated Macro Info

### PR DESCRIPTION
**Pull request by Akashneelesh**
I have merged it with the main branch and have updated the macro info part along with putting the example in a seperate folder

### CODE REVIEW

File: listings/.DS_Store
Comment (enitrat) [inline listings/.DS_Store:1]: what is this file

File: listings/ch11-advanced-features/.DS_Store
Comment (enitrat) [inline listings/ch11-advanced-features/.DS_Store:1]: what is this file

File: listings/ch99-starknet-smart-contracts/listing_99_09_call_contract_syscall/src/lib.cairo [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
     #[storage]
     struct Storage {}
 
     impl TokenWrapper of ITokenWrapper<ContractState> {
         fn transfer_token(
             ref self: ContractState,
-            address: starknet::ContractAddress,
-            selector: felt252,
-            calldata: Array<felt252>
+            address: ContractAddress,
```
Comment (enitrat) [inline listings/ch99-starknet-smart-contracts/listing_99_09_call_contract_syscall/src/lib.cairo:26]: I don't understand why I'm seeing changes between your branch and main here - it's not supposed to be the case. There must be something wrong with the way you managed your git history

**Comment by enitrat** [comment]
Please squash previous commits and then rebase on main. 
https://adamj.eu/tech/2022/03/25/how-to-squash-and-rebase-a-git-branch/

File: listings/ch99-starknet-smart-contracts/listing_99_09_call_contract_syscall/src/lib.cairo (hunk 1 above)
Comment (Akashneelesh) [reply listings/ch99-starknet-smart-contracts/listing_99_09_call_contract_syscall/src/lib.cairo]: Got it let me try rectifying it.

**Pull request closed without merging**